JustSaying/AwsTools/ErrorQueue.cs

```diff
@@ -25,7 +25,7 @@ public ErrorQueue(RegionEndpoint region, string sourceQueueName, IAmazonSQS clie
             return new Dictionary<string, string>
             {
                 { SQSConstants.ATTRIBUTE_MESSAGE_RETENTION_PERIOD, queueConfig.ErrorQueueRetentionPeriod.TotalSeconds.ToString("F0", CultureInfo.InvariantCulture)},
-                { SQSConstants.ATTRIBUTE_VISIBILITY_TIMEOUT, JustSayingConstants.DefaultVisibilityTimeout.ToString("F0", CultureInfo.InvariantCulture)},
+                { SQSConstants.ATTRIBUTE_VISIBILITY_TIMEOUT, JustSayingConstants.DefaultVisibilityTimeout.TotalSeconds.ToString("F0", CultureInfo.InvariantCulture)},
```

Comment:
Maybe we've done this `F0` call so many times across the code base now we should add an internal helper/extension method to do it?

Reply:
Agreed, done